scoda/templates/static/js/components/DataStoriesDetails.jsx

```diff
@@ -41,8 +41,8 @@ export default class DataStoriesDetails extends Component {
                     <div className="tab-pane fade show active" id="nav-sacn-content" role="tabpanel" aria-labelledby="nav-sacn-tab">
                       <div className="container-fluid">
                           <div className="row">
-                             <div className="col-md-1 col-lg-1 col-xl-1"></div>
-                             <div className="col-md-10 col-lg-10 col-xl-10">
+                             <div className=""></div>
+                             <div className="">
```

Comment:
@PhakamaniXulu if this div is not necessary, please remove. Otherwise if it's needed, but no classes are needed, please remove the classname attribute.